app/index.html

```diff
@@ -24,6 +24,8 @@
   <script type="text/javascript" src="js/livebus/livebus.factory.js"></script>
 
   <script type="text/javascript" src="js/shared/shared.module.js"></script>
+  <script type="text/javascript" src="js/shared/ucBackground/ucBackground.module.js"></script>
+  <script type="text/javascript" src="js/shared/ucBackground/ucBackground.directive.js"></script>
```

Comment:
What I meant with modular design wasn't really what Angular defines as modules. I do understand the way they are organized.

But good to hear we aren't that far off! If this was organized differently, just a few lines of code would be different. After minimizing the requests won't increase and the code isn't cluttered. I don't really see negatives.